### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AnyoneClown**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,25 +1,74 @@
...
 class IntegerRange:
-    pass
+
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(
+            self,
+            owner: SlideLimitationValidator,
```
Comment (Dimosphen1) [inline app/main.py]: Do not bind `owner` here and `instance`/`owner` from `__get__` to only the `SlideLimitationValidator` class, the `IntegerRange` class should be reusable across different classes. Replace `SlideLimitationValidator` with `Any`.

File: app/main.py [hunk 2]
```diff
@@ -1,25 +1,74 @@
...
+
+    def __set__(self, instance: object, value: int) -> None:
+        if value not in range(self.min_amount, self.max_amount + 1):
+            raise ValueError
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
```
Comment (Dimosphen1) [inline app/main.py]: Place all arguments in all `__init__` statements on a new line for better code perception and style.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

File: app/main.py (hunk 2 above)
Comment (pavlejviki) [reply app/main.py]: is not fixed

**Review by pavlejviki — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,25 +1,86 @@
...
+    def __get__(
+            self,
+            instance: Any,
+            owner: Any = None
+    ) -> None:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if value not in range(self.min_amount, self.max_amount + 1):
+            raise ValueError
```
Comment (viktoria-rybenchuk) [inline app/main.py]: write descriptive error meassage

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — APPROVED** [review]
Good job!

**Pull request closed without merging**